### PR TITLE
Zy/44 redesign login buttons

### PR DESCRIPTION
**Pull request by zheyxu**
For issue #44

### CODE REVIEW

File: analysis_options.yaml
Comment (gjwgit) [inline analysis_options.yaml:1]: Thanks. I was wondering if we should retain this or not.

**Review by gjwgit — APPROVED** [review]
Works great. `make prep` all good. Tested on Linux and Android. Thanks.

I made a couple of minor name changes: pod -> register. More informative?

Once @anushkavidanage has approved please merge it.

File: analysis_options.yaml
Comment (zheyxu) [reply analysis_options.yaml:1]: Ubuntu_lints has no rule of prefer_match_file_name. Thus, it should be removed.

Comment (gjwgit) [reply analysis_options.yaml:1]: I see. Thanks.

**Review by anushkavidanage — APPROVED** [review]
Reviewed and approved

**Comment by gjwgit** [comment]
Resolved conflict. Merged.

**Pull request merged**